nuget/chocolatey/tools/chocolateysetup.psm1

```diff
@@ -420,7 +420,7 @@ param(
 
   if (!(Test-Path("$chocoPkgDirectory\chocolatey.nupkg"))) {
     Write-Output "chocolatey.nupkg file not installed in lib.`n Attempting to locate it from bootstrapper."
-    $chocoZipFile = Join-Path $tempDir "chocolatey\chocInstall\chocolatey.zip"
+    $chocoZipFile = Join-Path $tempDir "chocolatey\chocoInstall\chocolatey.zip"

```

Comment:
This is probably not ok w/o some other change. Happy to drop.

Reply:
@gep13: Thanks, I think I've addressed everything else. I'm not realistically in a position to test this (or anything else), can I leave it to you?

If you'd like it split into its own thing, I can do that as well.